### PR TITLE
feat: make tag name monolingual

### PR DESCRIPTION
**Pull request by winged**
There are several reasons for this change:

* Tags may only make sense in one of several languages in use.  There is
  no guarantee that every tag will have a suitable translation into all
  other languages either.

* Tag creation is in almost all tagging systems an "ad-hoc" sort of
  work. There is no centralized management of tags, where an admin would
  sift though the tags, translate them, etc

* There is no way to guarantee that a tag created by user A in their
  language will be usable by user B, if they use another language.

The way forward with this is to add a "language" field, so tags can
be filtered to only show the ones given in your current language.
However the exact desired behaviour in this regard is not yet clear,
so we're leaving out this field for now.

So at this point in time, tags are a monolingual feature. We'll need
to determine their beheviour once we have a multilingual application.

cc @fkm-adfinis

### CODE REVIEW

File: alexandria/core/serializers.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def create(self, validated_data):
         slug_field = getattr(self.Meta, "slug_source_field", "name")
         if "slug" not in validated_data:
             slug_source_value = validated_data.get(slug_field)
-            if isinstance(slug_source_value, dict):
+            if isinstance(slug_source_value, dict):  # pragma: no cover
```
Comment (open-dynaMIX) [inline alexandria/core/serializers.py:73]: I'd prefer a `pragma: todo cover` here.

**Review by open-dynaMIX — APPROVED** [review]
:tada:

**Comment by winged** [comment]
@fkm-adfinis please implement the corresponding change in ember-alexandria :)

**Comment by fkm-adfinis** [comment]
:sunglasses: 

I'll update the addon.

**Pull request merged**